messaging/MessagingExample/AppDelegate.m

```diff
@@ -125,6 +125,8 @@ - (void)application:(UIApplication *)application didReceiveRemoteNotification:(N
 
   // Print full message.
   NSLog(@"%@", userInfo);
+
+  completionHandler(UIBackgroundFetchResultNewData);
```

Comment:
@kroikie, Any specific reason for using this instead of `UIBackgroundFetchResultNoData`?